src/lib/storage.js

```diff
@@ -68,12 +68,20 @@ async function createDynamoItem({ table, keyName, keyValue, data = {}, logger })
 async function batchWriteDynamoItems({ batch, logger }) {
   try {
     telemetry.increaseCount('dynamo-batch-inserts')
-    const request = composeBatchInsert(batch)
-    const command = new BatchWriteItemCommand(request)
-    const response = await telemetry.trackDuration('dynamo-batch-inserts', sendCommand({ client: dynamoClient, command, logger }))
-    if (response?.UnprocessedItems && Object.keys(response.UnprocessedItems).length > 0) {
-      logger.warn({ UnprocessedItems: JSON.stringify(response.UnprocessedItems) }, 'UnprocessedItems in BatchWriteItemCommand')
-    }
+    let request = composeBatchInsert(batch)
+    let done
+    do {
+      const command = new BatchWriteItemCommand(request)
+      const response = await telemetry.trackDuration('dynamo-batch-inserts', sendCommand({ client: dynamoClient, command, logger }))
+      if (response?.UnprocessedItems && Object.keys(response.UnprocessedItems).length > 0) {
+        telemetry.increaseCount('dynamo-unprocessed-items')
+        logger.warn({ UnprocessedItems: JSON.stringify(response.UnprocessedItems) }, 'UnprocessedItems in BatchWriteItemCommand')
```

Comment:
Question: Do we want to consider this a `warn` or an `info`?  If AWS treats this as a normal occurrence, then we may as well also.